src/playground-ui/ConfirmDialog.tsx

```diff
@@ -8,10 +8,12 @@ import DialogTitle from "@material-ui/core/DialogTitle";
 import TextField from "@material-ui/core/TextField";
 import React, { useState } from "react";
 
+export type ConfirmValue  = "undefined" | "load" | "replace" | "nevermind"
```

Comment:
All of the generics in this and `ConfirmDialogProvider` were around taking these `value` values and making sure that they were tracked by the typesystem. I got to a place where typescript was complaining that two apparently-identical types were distinct and therefore not assignable to each other. I made the decision that it wasn't worth the complexity, in favor of explicitly typing the `ConfirmValue`.

The downside of this approach is that this enumeration needs to change every time that a new desired value for `value` is added, but given the way we're matching on the values, that's probably okay.

All other changes in these files are propagating this change.